### PR TITLE
@zephraph => [Lib] Add @artsy/palette

### PR DESCRIPTION
**Pull request by damassi**
This PR adds [`@artsy/palette`](https://github.com/artsy/palette) into the mix.

### CODE REVIEW

File: src/Styleguide/Elements/__stories__/Typography.story.tsx [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
+  })
   .add(
     "Sans",
     withInfo(`
 
     https://www.notion.so/artsy/Typography-d1f9f6731f3d47c78003d6d016c30221
 
   `)(() => {
+      // prettier-ignore
+      const sizes = ['1', '2', '3', '3t', '4', '4t', '5',
```
Comment (zephraph) [inline src/Styleguide/Elements/__stories__/Typography.story.tsx]: Why hard code the sizes? We could just read these from the `theme`, right?

Comment (damassi) [reply src/Styleguide/Elements/__stories__/Typography.story.tsx]: Oh right :doh:

File: src/Styleguide/Elements/__stories__/Typography.story.tsx [hunk 2]
```diff
@@ -1,38 +1,35 @@
...
+import { Col } from "../Grid"
 import { withInfo } from "@storybook/addon-info"
-import { Display, Sans, Serif } from "../Typography"
 
 const stories = storiesOf("Styleguide/Typography", module)
 
 stories
+  .addDecorator(storyFn => {
+    return <Theme>{storyFn()}</Theme>
+  })
```
Comment (alloy) [inline src/Styleguide/Elements/__stories__/Typography.story.tsx]: Could this go into our own storiesOf wrapper?

Comment (zephraph) [reply src/Styleguide/Elements/__stories__/Typography.story.tsx]: Oh, yeah! `storiesOf("Styleguide")`? Would that apply to even the nested stories?

Comment (damassi) [reply src/Styleguide/Elements/__stories__/Typography.story.tsx]: I'll give it a try

Comment (alloy) [reply src/Styleguide/Elements/__stories__/Typography.story.tsx]: I mean something like:

```js
function storiesOf(desc, mod) {
  return require(“storybooks”).storiesOf(desc, mod).addDecorator(...)
}
```

File: src/Styleguide/Artwork/__stories__/Banner.story.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import { storiesOf } from "@storybook/react"
+import { storiesOf } from "storybook/storiesOf"
```
Comment (damassi) [inline src/Styleguide/Artwork/__stories__/Banner.story.tsx]: @alloy - your suggestion worked great. What I was a bit unsure of was where to put it. As more stuff comes online i figured that we would be using it more so I stashed it in `src/__stories__` next to the config file and created this alias, but if it makes more sense just to keep it inside of `Styleguide` i can move it out.

**Comment by zephraph** [comment]
Seems to be several build failures... oddly, they all seem related to timestamps.

File: src/Styleguide/Elements/__stories__/Typography.story.tsx [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
 
 const stories = storiesOf("Styleguide/Typography", module)
 
+function getTypeSizes(type) {
+  const sizes = Object.keys(themeProps.typeSizes)
+    .filter(size => size.includes(type))
+    .map(size => size.replace(type, ""))
+
+  return sizes
+}
```
Comment (zephraph) [inline src/Styleguide/Elements/__stories__/Typography.story.tsx]: Nice!

**Comment by zephraph** [comment]
Other than the CI issue, LGTM!

**Comment by damassi** [comment]
(Going to merge since just date failures)

File: src/Styleguide/Artwork/__stories__/Banner.story.tsx (hunk 3 above)
Comment (alloy) [reply src/Styleguide/Artwork/__stories__/Banner.story.tsx]: Nice 😎

**Pull request merged**